docs/index.rst

```diff
@@ -134,7 +130,10 @@ Some basic usage::
 
     testimony validate tests/
 
-For further information about the options check Testimony's help.
+Misc:
```

Comment:
Add a blank line below, I think the way it is Sphinx will not generate a list properly. Also you can remove indention of the beginning of each list item.